frameworks/PHP/fat-free/index.php

```diff
@@ -50,6 +50,7 @@ function ($f3) {
 $f3->route(
     array(
         'GET /db-multiple',
+        'GET /db-multiple/',
```

Comment:
Duplicate with the / suffix don't look a good idea.
Delete the other one without /

In the benchmark_config.json end with /.

Reply:
Fat free doesn't assume a backslash will be appended for the route to work. I know it looks dumb, but for whatever reason the benchmark seems to be hitting `/db-multiple` and `/db-multiple/` for something and also checking `/db-multiple/123` I couldn't figure out another reason why this would happen https://tfb-status.techempower.com/unzip/results.2020-04-20-07-27-56-665.zip/results/20200415180849/fat-free/update/verification.txt on a url that's been defined in the benchmark_config.json